BrickKit/bricks/src/main/java/com/wayfair/brickkit/BrickViewHolder.java

```diff
@@ -12,6 +12,7 @@
  * one additional method that is called when the view holder is detached from the {@link RecyclerView} so
  * that views can be released.
  */
+//TODO: migrate this class to the view holder package
```

Comment:
I put a note here, but held off on it to mitigate migration issues until everyone thinks it is a good time to migrate it to the other package.

Reply:
🤷‍♂ I'd just move it :-)

Reply:
I refactored it @kenyee